### ISSUE TITLE
Outdated tutorial

### ISSUE THREAD

**Issue by arnau** [Marwes/combine#339]
Hi, I started following the [tutorial](https://github.com/Marwes/combine/wiki/Tutorial#code-structure) and taking the lazy approach and copying the example verbatim fails with a couple errors including one where `state::State` does not exist.

Is this a matter of updating the tutorial to a new version of 'combine' or am I missing a step to make it work?

**Comment by Marwes** [comment]
Yeah, it needs updating. probably just some import paths that have changed. Since it is on the github wiki it isn't tested automatically.
